lib/proxywrap.js

```diff
@@ -290,5 +281,21 @@ exports.proxy = function(iface, options) {
 
 	}
 
+	function createSecureConnectionListener(context, listener) {
+		return function (socket) {
+			var properties = Object.keys(proxyProtocolFields);
+			defineTLSSocketProperties(socket, properties);
+			listener.call(context, socket);
+		}
+	}
+
+	function defineTLSSocketProperties(tlsSocket, properties) {
+		for(var i in properties) {
```

Comment:
Can you add test units for this?

I believe there is some [work already made](https://github.com/findhit/proxywrap/blob/master/test/proxy.protocol.v1.test.js#L24-L29). Just uncomment it and give it a try.